ksql-common/src/main/java/io/confluent/ksql/util/ReservedInternalTopics.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
+
+public final class ReservedInternalTopics {
+  private static final Set<String> ALL_INTERNAL_TOPICS_PREFIXES = ImmutableSet.of(
+      // Confluent
+      "_confluent",
+      "__confluent"
+  );
```

Comment:
The processing log topic is not hidden from users. This topic is considered a non-internal, that's why the processing log topic `default_ksql_processing_log` does not have the prefix for an internal topic.

Reply:
Done. I did not include the processing in the list of topics to hide because users are intentionally meant to read from this topic and should know what name it is. I did add a check to avoid writing to it, though.